playbook/app/pb_kits/playbook/pb_background/docs/style.scss

```diff
@@ -0,0 +1,4 @@
+
+.classy {
+  border: 1rem solid green;
+}
```

Comment:
Are you wanting to keep this in production?

Reply:
No, I can delete this.